SConstruct

```diff
@@ -1233,6 +1233,13 @@ python_min_version = LooseVersion('3.5')
 cython_min_version = LooseVersion('0.23')
 numpy_min_test_version = LooseVersion('1.8.1')
 
+# We choose ruamel.yaml 0.15.34 as the minimum version
+# since it is the highest version available in the Ubuntu
+# 18.04 repositories and seems to work. Older versions such as
+# 0.13.14 on CentOS and 0.10.23 on Ubuntu 16.04 raise an exception
```

Comment:
This is actually only for CentOS7. CentOS8 has 0.15.41 and will work.

Reply:
Thanks for the note! I added it in the most recent update.